task5/classes/Page.java

```diff
@@ -0,0 +1,14 @@
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+@WebServlet("/page")
+public class Page extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException, ServletException {
+        request.getRequestDispatcher("/WEB-INF/page.html").forward(request, response);
```

Comment:
Also, from the clean code
It's required to  declare all "string literals" as constants
```
private static final String STARTING_PAGE_ADDRESS = "/WEB-INF/page.html";
//....
request.getRequestDispatcher(STARTING_PAGE_ADDRESS ) // ...
```
Constants are situated immediately at the beginning of a class